### PR TITLE
switch to slack contact details

### PR DESCRIPTION
**Pull request by mashehu**
is the slack icon + `@` too much?
Screenshot 2022-11-04 at 11 23 41

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1430"></a>

### CODE REVIEW

**Review by JoseEspinosa — APPROVED** [review]
LGTM 🚀

File: markdown/events/2023/hackathon-march-2023.md [hunk 1]
```diff
@@ -25,48 +25,48 @@ Once sat together, join the global gathering online, on gather.town together.
 
 Watch this space for more details in the coming weeks!
 
+### Denmark
+
+- Unseen Bio ApS, Copenhagen
+  - Contact: Moritz Beber [<i class="fab fa-slack"></i> @moritz](https://nfcore.slack.com/team/U015PB15C5U)
```
Comment (ewels) [inline markdown/events/2023/hackathon-march-2023.md:31]:
```suggestion
  - Contact: Moritz Beber [<i class="fab fa-slack"></i> Moritz Beber](https://nfcore.slack.com/team/U015PB15C5U)
```

File: markdown/events/2023/hackathon-march-2023.md [hunk 2]
```diff
@@ -25,48 +25,48 @@ Once sat together, join the global gathering online, on gather.town together.
...
+  - _Details to be confirmed; based on interest_
+
 ### Germany
 
 - QBiC, Tübingen
-  - Contact: [@ggabernet](https://github.com/ggabernet/)
+  - Contact: [<i class="fab fa-slack"></i> @Gisela Gabernet](https://nfcore.slack.com/team/UG3J99VUK)
```
Comment (ewels) [inline markdown/events/2023/hackathon-march-2023.md:37]:
```suggestion
  - Contact: [<i class="fab fa-slack"></i> Gisela Gabernet](https://nfcore.slack.com/team/UG3J99VUK)
```

File: markdown/events/2023/hackathon-march-2023.md [hunk 3]
```diff
@@ -25,48 +25,48 @@ Once sat together, join the global gathering online, on gather.town together.
...
   - _Details to be confirmed_
 - MPI-EVA, Leipzig
-  - Contact: [@jfy133](https://github.com/jfy133/)
+  - Contact: [<i class="fab fa-slack"></i> @James Fellows Yates](https://nfcore.slack.com/team/UEM37TBAR)
```
Comment (ewels) [inline markdown/events/2023/hackathon-march-2023.md:40]:
```suggestion
  - Contact: [<i class="fab fa-slack"></i> James Fellows Yates](https://nfcore.slack.com/team/UEM37TBAR)
```

File: markdown/events/2023/hackathon-march-2023.md [hunk 4]
```diff
@@ -25,48 +25,48 @@ Once sat together, join the global gathering online, on gather.town together.
...
   - _Details to be confirmed_
 
 ### Spain
 
 - Centre for Genomic Regulation (CRG), Barcelona
-  - Contact: [@joseespinosa](https://github.com/joseespinosa/)
+  - Contact: [<i class="fab fa-slack"></i> @Jose Espinosa-Carrasco](https://nfcore.slack.com/team/UT94DM516)
```
Comment (ewels) [inline markdown/events/2023/hackathon-march-2023.md:46]:
```suggestion
  - Contact: [<i class="fab fa-slack"></i> Jose Espinosa-Carrasco](https://nfcore.slack.com/team/UT94DM516)
```

File: markdown/events/2023/hackathon-march-2023.md [hunk 5]
```diff
@@ -25,48 +25,48 @@ Once sat together, join the global gathering online, on gather.town together.
...
   - _Details to be confirmed_
 
 ### Sweden
 
 - SciLifeLab, Stockholm
-  - Contact: [@FranBonath](https://github.com/FranBonath/)
+  - Contact: [<i class="fab fa-slack"></i> @Franziska Bonath](https://nfcore.slack.com/team/UGP9YUCKD)
```
Comment (ewels) [inline markdown/events/2023/hackathon-march-2023.md:52]:
```suggestion
  - Contact: [<i class="fab fa-slack"></i> Franziska Bonath](https://nfcore.slack.com/team/UGP9YUCKD)
```

File: markdown/events/2023/hackathon-march-2023.md [hunk 6]
```diff
@@ -25,48 +25,48 @@ Once sat together, join the global gathering online, on gather.town together.
...
   - _Details to be confirmed_
 
 ### United Kingdom
 
 - London
-  - Contact: [@drpatelh](https://github.com/drpatelh/)
+  - Contact: [<i class="fab fa-slack"></i> @hpatel](https://nfcore.slack.com/team/UEB97FBN3)
```
Comment (ewels) [inline markdown/events/2023/hackathon-march-2023.md:58]:
```suggestion
  - Contact: [<i class="fab fa-slack"></i> Harshil Patel](https://nfcore.slack.com/team/UEB97FBN3)
```

File: markdown/events/2023/hackathon-march-2023.md [hunk 7]
```diff
@@ -25,48 +25,48 @@ Once sat together, join the global gathering online, on gather.town together.
...
   - _Details to be confirmed_
 
 ### United States
 
 - University of Texas at Dallas, Texas
-  - Contact: [@Emiller88](https://github.com/Emiller88/)
+  - Contact: [<i class="fab fa-slack"></i> @Edmund Miller](https://nfcore.slack.com/team/UV41DBFAT)
```
Comment (ewels) [inline markdown/events/2023/hackathon-march-2023.md:64]:
```suggestion
  - Contact: [<i class="fab fa-slack"></i> Edmund Miller](https://nfcore.slack.com/team/UV41DBFAT)
```

File: markdown/events/2023/hackathon-march-2023.md [hunk 8]
```diff
@@ -25,48 +25,48 @@ Once sat together, join the global gathering online, on gather.town together.
...
   - _Details to be confirmed_
 - The University of Wyoming
-  - Contact: [@rpetit3](https://github.com/rpetit3/)
+  - Contact: [<i class="fab fa-slack"></i> @Robert A. Petit III](https://nfcore.slack.com/team/U01PGBF4P0R)
```
Comment (ewels) [inline markdown/events/2023/hackathon-march-2023.md:67]:
```suggestion
  - Contact: [<i class="fab fa-slack"></i> Robert A. Petit III](https://nfcore.slack.com/team/U01PGBF4P0R)
```

**Review by ewels — APPROVED** [review]
Slack icon is great, but personally I think that the `@` is a little redundant and makes it a bit difficult to read.

Also swapped out a couple of usernames -> names in suggested changes.

File: markdown/events/2023/hackathon-march-2023.md (hunk 1 above)
Comment (mashehu) [inline markdown/events/2023/hackathon-march-2023.md:31]:
```suggestion
  - Contact: [<i class="fab fa-slack"></i> Moritz Beber](https://nfcore.slack.com/team/U015PB15C5U)
```

**Pull request merged**